task/ecosystem-cert-preflight-checks/0.1/ecosystem-cert-preflight-checks.yaml

```diff
@@ -25,7 +25,7 @@ spec:
           mountPath: /artifacts
       script: |
         # Expected results directory
-        RESULT_JSON_PATH=artifacts/results.json
+        RESULT_JSON_PATH=artifacts/amd64/results.json
```

Comment:
Under what conditions will this architecture change?

Reply:
amd64 is the default architecture so this should not change. Additional folders will be created in case preflight is provided a multi-arch image ex arm64 but my understanding is that there will always be a default amd64 folder.

Reply:
Instead of hard coding this, would it be possible to inspect the image to see what architectures are present? It would be preferable if we don't need to modify the task to increase the supported architectures. We already support building amd64, arm64, ppc64le, and s390x architectures. While it might be a reasonable assumption that amd64 is always present, that isn't necessarily a requirement.

Reply:
agreed 👍  Will change it to process results from all architectures if present.